debezium-server/debezium-server-pubsub/src/main/java/io/debezium/server/pubsub/PubSubChangeConsumer.java

```diff
@@ -141,10 +157,20 @@ void connect() {
                     .setLimitExceededBehavior(FlowController.LimitExceededBehavior.Block)
                     .build());
         }
+        
+        if (address.isPresent()) {
+        	String hostport = address.get();
+        	channel = ManagedChannelBuilder
+        			.forTarget(hostport)
+        			.usePlaintext()
+        			.build();
+        	channelProvider = FixedTransportChannelProvider.create(GrpcTransportChannel.create(channel));
+        	credentialsProvider = NoCredentialsProvider.create();        	
```

Comment:
I wonder if it might be necessary to override the credentials provider.

Reply:
That's how it's done in the pub sub example: https://cloud.google.com/pubsub/docs/emulator#accessing_environment_variables